src/Petabridge.Library/Petabridge.Library.csproj

```diff
@@ -2,7 +2,11 @@
 
   <PropertyGroup>
```

Comment:
Reviews starts from here. Branched carried with it commits from the master branch - still struggling with `git` :)